browser/mapping.go

```diff
@@ -616,16 +616,21 @@ func mapWorker(vu moduleVU, w *common.Worker) mapping {
 func mapBrowserContext(vu moduleVU, bc *common.BrowserContext) mapping { //nolint:funlen
 	rt := vu.Runtime()
 	return mapping{
-		"addCookies":                  bc.AddCookies,
-		"addInitScript":               bc.AddInitScript,
-		"browser":                     bc.Browser,
-		"clearCookies":                bc.ClearCookies,
-		"clearPermissions":            bc.ClearPermissions,
-		"close":                       bc.Close,
-		"cookies":                     bc.Cookies,
-		"exposeBinding":               bc.ExposeBinding,
-		"exposeFunction":              bc.ExposeFunction,
-		"grantPermissions":            bc.GrantPermissions,
+		"addCookies":       bc.AddCookies,
+		"addInitScript":    bc.AddInitScript,
+		"browser":          bc.Browser,
+		"clearCookies":     bc.ClearCookies,
+		"clearPermissions": bc.ClearPermissions,
+		"close":            bc.Close,
+		"cookies":          bc.Cookies,
+		"exposeBinding":    bc.ExposeBinding,
+		"exposeFunction":   bc.ExposeFunction,
+		"grantPermissions": func(permissions []string, opts goja.Value) error {
+			parsedOpts := common.NewGrantPermissionsOptions()
```

Comment:
I don't mind, and this is bikeshedding (😅), but it's more idiomatic to use short variable names within short contexts. Instead of `parsedOpts`, we can just say something like `popts`. Also, before parsing, semantically, `parsedOpts` are not parsed yet.

Reply:
Yeah, makes sense 👍 I've renamed it to `pOpts`, since they will be parsed on the next line and we can't change the name of the variable after it's been parsed 🙂 

Resolved in 74822b1ddc3524ca37fe80cafeef623bcd0b8b75

Reply:
To be honest, I also say `popts` everytime 😄